src/state/application/reducer.ts

```diff
@@ -41,7 +41,7 @@ export default createReducer(initialState, (builder) =>
       if (chainId === ALL_CHAIN_ID) {
         return {
           ...state,
-          activeNetworksId: ALL_SUPPORT_NETWORKS_ID,
+          activeNetworksId: Object.keys(NETWORKS_INFO_MAP).map(Number),
```

Comment:
This is bad. We should have constant for things like this

Reply:
anyway, I think I can use SUPPORTED_NETWORKS on that place.

Reply:
<img width="797" alt="image" src="https://user-images.githubusercontent.com/33005392/181416101-722df377-3e47-4b9b-a8ac-0be0909e97b5.png">
like this

Reply:
we gonna use `ELASTIC_SUPPORTED_NETWORKS` as I proposed

Reply:
<img width="789" alt="image" src="https://user-images.githubusercontent.com/33005392/181418570-e72648b5-fe7e-4abc-bc8d-6825cad366b5.png">
all constant now. 
=> on that I will use **SUPPORTED_NETWORKS**

- CLASSIC_SUPPORTED_NETWORKS, ELASTIC_SUPPORTED_NETWORKS will used for rendering menu
- SUPPORTED_NETWORKS: contain all chain.  (filter not support network. ChainId.ROPSTEN, ChainId.RINKEBY, ChainId.AURORA no need to include here, because we don't render it on UI, don't calc data on them)
- ALL_SUPPORTED_NETWORKS = SUPPORTED_NETWORKS+ALL_CHAIN_ID

is it ok

Reply:
No no, we dont render them on menu, but still supported it when we type address directly to the url

Reply:
It just like ks, dont show them on menu but support them

Reply:
For aurora, it's a special case, but we dont need to separated it a new constant without it. Just catch the url and redirect from it to /home.